solutions/usaco-893.mdx

```diff
@@ -0,0 +1,123 @@
+---
+id: usaco-893
+source: USACO Bronze 2019 January
+title: Guess the Animal
+author: Brad Ma
+---
+
+[Official Analysis (C++)](https://codeforces.com/blog/entry/22712)
+
+<LanguageSection>
+  
+<CPPSection>
+  
+```cpp
+#include <iostream>
+#include <fstream>
+#include <vector>
+using namespace std;
+ 
+int N;
+vector<string> characteristics[100];
+
+// Official Analysis Code by Brian Dean
+ 
+int num_common(int i, int j)
+{
+	int count = 0;
+	vector<string> &v1 = characteristics[i], &v2 = characteristics[j];
+	for (int i=0; i<v1.size(); i++)
+	for (int j=0; j<v2.size(); j++)
+		if (v1[i] == v2[j]) count++;
+	return count;
+}
+ 
+int main(void)
+{
+	ifstream fin ("guess.in");
+	fin >> N;
+	string s;
+	for (int i=0; i<N; i++) {
+	int K;
+	fin >> s >> K;
+	for (int j=0; j<K; j++) {
```

Comment:
could you edit this code and add spaces around all the operators? like
```suggestion
	for (int j = 0; j < K; j++) {
```

Reply:
^ not addressed